SwrveSDK/SDK/Track/Swrve.m

```diff
@@ -1417,6 +1449,8 @@ -(void) shutdown
     }
 
     [self setEventBuffer:nil];
+    [[self eventStream] removeFromRunLoop:[NSRunLoop currentRunLoop]
```

Comment:
This is to ensure it's no longer running after shutdown : 
https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Streams/Articles/ReadingInputStreams.html